Data-Structures/Heap/test/KeyPriorityQueue.test.js

```diff
@@ -0,0 +1,124 @@
+import { KeyPriorityQueue } from '../KeyPriorityQueue.js'
+
```

Comment:
You should wrap your tests in one big `describe("Key Priority Queue", () => { ... })`.